### PR TITLE
Correction of Laminas\Code\Generator\DocBlock\Tag\ParamTag example

### PR DESCRIPTION
**Pull request by trippodi**
1st Parameter for Laminas\Code\Generator\DocBlock\Tag\ParamTag::_construct() does not accept an array. Also: Adding example that parameter 'types" may also be array (e.g. 'string' and 'array') and contain a description.
* Documentation improvement: master branch

|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | yes
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | no

### Description
There's an error in the given example: The constructor for Laminas\Code\Generator\DocBlock\Tag\ParamTag is not a single array. Also extended the example by using an array for parameter 'types' and adding the 'description' parameter.

### CODE REVIEW

File: docs/book/generator/examples.md [hunk 1]
```diff
@@ -153,10 +153,14 @@ $foo->setName('Foo')
...
-                        'datatype'  => 'string'
-                    ]),
+                     new Tag\ParamTag(
+                        'bar', 
+                        [
+                            'string',
+                            'array'
+                        ],
+                        'parameter description'
+                        )
```
Comment (Ocramius) [inline docs/book/generator/examples.md:163]: `),` (comma needed here), and need to fix some alignment

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by trippodi** [comment]
I use Git & Github very rarely. Could you kindly hint me on how to proceed with implementing your corrections on my fork so it meets your requirements .

**Comment by Ocramius** [comment]
@mdthh sorry, totally missed to follow up here: I'll get your branch fixed and merged locally :-)

For future reference, when/if it comes up, you can clone your fork to any local directory, edit the files on the branch you were working on, then push that branch, and the pull request will be updated implicitly.

**Comment by Ocramius** [comment]
Rebased and merged manually, thanks @mdthh!

**Pull request closed without merging**